### DISCUSSION TITLE
Только треки с кириллицей

### DISCUSSION THREAD

**Discussion by ghost** [Chimildic/goofy#133] [Помощь с алгоритмом] [answered]
Мне нужно, чтобы в плейлист добавлялись только те треки, чье название на русском языке или в чьём названии присутствует хотя бы буква/слово на русском языке. Как я могу это сделать?

**Comment by Chimildic** [comment, marked as answer]
```js
Filter.match(tracks, '[а-яА-ЯёЁ]+');
```
